.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: membrane/bionic-membrane:latest
+      - image: membrane/membrane:latest
```

Comment:
You can also add a step checking for compilation warnings

Reply:
done